pkg/volume/csi/csi_attacher.go

```diff
@@ -66,7 +66,7 @@ func (c *csiAttacher) Attach(spec *volume.Spec, nodeName types.NodeName) (string
 
 	pvSrc, err := getPVSourceFromSpec(spec)
 	if err != nil {
-		klog.Error(log("attacher.Attach failed to get CSIPersistentVolumeSource: %v", err))
+		klog.Errorf(log("attacher.Attach failed to get CSIPersistentVolumeSource: %v", err))
```

Comment:
`log()` prepares the log statement using parameterized values already.  There no need for `klog.Errorf` or `klog.Infof`.

This goes for all other occurrences in this PR.